sdk/python/feast/templates/gcp/feature_store.yaml

```diff
@@ -1,3 +1,4 @@
 project: my_project
-registry: data/registry.db
+registry:
```

Comment:
It's not clear why we are making changes here. The default string based option is meant to be a concise way to specify a storage location.

Reply:
That's a good point, I was thinking we were going to change the default to specify the metadata folder path instead of the filepath location. But maybe it could be a good design to allow `registry: <path>` accept both filepath or dirpath and just continue to support both syntaxes (single line for just specifying path and multiline for nested options).